MicroPython/src/communication/API_communication.py

```diff
@@ -20,30 +23,34 @@ def authorization_request() -> str:
     try:
         response = urequests.post(url, data=body, headers=headers)
     except IndexError as e:
-        logging.ingo("No internet connection")
-        return None
+        logging.info("No internet connection: {}".format(e))
+        return ""
     except Exception as e:
         logging.info("Failed to authorize in API {}".format(e))
-        return None
+        return ""
 
     if response.status_code != '200' and response.status_code != 200:
         logging.error(response.text)
-        return None
+        return ""
 
     response_dict = response.json()
     jwt_token = response_dict.get("data")
     return jwt_token
 
 
 def configuration_request(_jwt_token: str):
+    """
+    Function configures ESP with cloud (AWS)
+    :param _jwt_token: JSON Web Token
+    :return: dict with certificates/keys
+    """
     headers = get_header_with_authorization(_jwt_token)
     url = config.cfg.api_url + config.API_CONFIG_URL
     thing_name = config.THING_NAME_PREFIX + config.cfg.device_uid
     body = {}
     body['is_removed'] = True
     body['created_at'] = 0
     body['device_id'] = thing_name
-    #TODO what values should be below?
     body['description'] = 'Full configuration test'
```

Comment:
Please create variables in config for default description, device group and type